pkg/secrets/k8s_secrets_storage/mocks/k8s_secrets_client.go

```diff
@@ -17,17 +18,18 @@ func RetrieveK8sSecret(_ string, secretName string) (map[string][]byte, error) {
 		return nil, errors.New("custom error")
 	}
 
-	return MockK8sDB[secretName], nil
+	return &v1.Secret{
+		Data: MockK8sDB[secretName]}, nil
```

Comment:
please change to:
```
&v1.Secret{
  Data: MockK8sDB[secretName]
}, nil
```

Reply:
Fmt is not happy about this (Need trailing comma before newline in composite literal), but can do this with a comma if you prefer
```
	return &v1.Secret{
		Data: MockK8sDB[secretName],
	}, nil
```

Reply:
Yes, that will do. thanks.